gm4_metallurgy/data/metallurgy/functions/blasting/tnt.mcfunction

```diff
@@ -0,0 +1,6 @@
+# @s = primed tnt with a fuse of 1
+# run from main
```

Comment:
This says it is being ran from main but it is actually ran from [pulse check](https://github.com/Gamemode4Dev/GM4_Datapacks/blob/19cf7d44b71732175ba12769822df6f1b213fc47/gm4_metallurgy/data/metallurgy/functions/pulse_check.mcfunction#L6)

Reply:
Resolved in next commit.